packages/xo-server-sdn-controller/src/index.js

```diff
@@ -665,6 +909,53 @@ class SDNController extends EventEmitter {
       pool: network.$pool.name_label,
     })
 
+    // If the network is cross-pool: reconnect to other networks
+    if (crossPoolNetwork !== undefined) {
+      const centerPoolNetwork = this._getPoolNetwork(
+        crossPoolNetwork.poolCenter,
+        crossPoolNetwork
+      )
+      if (crossPoolNetwork.poolCenter === newCenter.$pool.$ref) {
```

Comment:
IMO it should be clear that `newCenter` is a host, especially since "center" is also used for pools now.

Reply:
What does the thumb-up mean?

Reply:
I agree with the comment. I'm working on a DEEP refactor of the plugin to separate pool-wide and cross-pool networks in different objects and specific methods to simplify and clarify the implmentation.

Reply:
Ok, @pdonias will love the review :rofl: